src/components/JobListing.jsx

```diff
@@ -172,11 +177,17 @@ const JobListing = ({ rowIndex, handleClaimJob, ...props }) => {
                         </button>
                     )
                 ) : (
-                    <span className='text-red-700 text-center'>
-                        Filled
-                        <br />
-                        {props.FillDate ? `${formatDate(props.FillDate)}` : ''}
-                    </span>
+                    props.open == 'Filled' ? (
+                        <span className='text-red-700 text-center'>
+                            Filled
+                            <br />
+                            {props.FillDate ? `${formatDate(props.FillDate)}` : ''}
+                        </span>
+                    ) : (
+                        <span className='text-red-700 text-center'>
+                            Filled by CO
```

Comment:
I think we should still include the date

```suggestion
                            Filled by CO
                            <br />
                            {props.FillDate ? `${formatDate(props.FillDate)}` : ''}
```